src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2147,6 +2147,141 @@ It is possible to restore any previous commit to the HEAD position by clicking t
+
+The figure below shows how the user visualize the authorization icon next to the bucket.
+
+image::../images/user-bucket-grant-icon.png[align=center]
+
+The figure below shows how the user visualize the authorization icon next to the objects.
+
+image::../images/user-object-grant-icon.png[align=center]
+
+If the object does not have an icon next to it (e.g. `Native_Task_Windows`), it implies that it has not a direct authorization assign to it, and the user will have the same right over it as she/he has over the bucket containing it.
+
+To delete an authorization, the admin simply needs to select one from the list and the delete button will be activated.
+It is the button with a "bin" symbol next to the add a new authorization button "+".
+The figure below shows an example.
+
+image::../images/DeleteGrant.png[align=center]
+
+Once the admin delete the authorization, it will be removed from the list as shown in the figure below.
+
+image::../images/DeletedGrant.png[align=center]
+
+To update an authorization, the admin needs to click on the drop-down list of the access type or the priority level and select a new value.
+The figure below shows an example.
+
+image::../images/updateAGrant.png[align=center]
+
+Once the selection is made the authorization will be updated as shown in the figure below.
+
+image::../images/updatedGrant.png[align=center]
+
+However, when a user has admin rights over a bucket or an object, he/she can not downgrade his/her rights while updating the authorization.
+Another user, with admin rights over the bucket or the object, can do it for him/her.
+
+==== Authorization calculation rules
+
+Since each user might belong to multiple user groups, a user could have multiple authorizations over a bucket due to his/her username or user group(s) authorizations.
+In such case, the resulting access type will be calculated as follows:
+
+* If the username-assigned authorization exists, it is prioritized and its access type will be the user's resulting rights over the bucket.
+
+* If multiple user-groups authorizations exist, without a username authorization, the resulting user's rights over the bucket will be the access type of the group authorization that have the highest priority.
+
+In the case where a user has multiple authorizations over an object, the resulting access type will be calculated as follows:
+
+* If the username-assigned authorization for the object exists, it is prioritized and its access type will be the user's resulting rights over the object.
+
+* If multiple user-groups authorizations exist for the object, without a username authorization, the resulting user's rights over the object will be the access type of the group authorization that have the highest priority.
+
+* If both username-assigned authorization and user-groups authorizations do not exist for the object, the resulting user's rights for the object will be the same as the user's resulting rights over the bucket that contains the object.
```

Comment:
If both username-assigned authorization**s** and user-groups authorizations
(please use plural tense for homogeneity)

Reply:
I'll update it. Thanks.